lib/index.js

```diff
@@ -128,18 +134,25 @@ util.inherits(SpawnMocha, EventEmitter);
 
 var mochaStream = function mocha(opts) {
   opts = opts || {};
+  var files = [];
```

Comment:
This PR is to base concurrency on the `iterations` Array elements as an alternative to file concurrency. Are you concerned that waiting for the `gulp.src` stream to end before spawning the mocha processes is going to be a performance drag?